tests/integration/integration_nodejs_test.go

```diff
@@ -1017,3 +1017,35 @@ func TestComponentProviderSchemaNode(t *testing.T) {
 	}
 	testComponentProviderSchema(t, path)
 }
+
+// Test throwing an error within an apply in a remote component written in nodejs.
+// The provider should return the error and shutdown gracefully rather than hanging.
+func TestConstructNodeErrorApply(t *testing.T) {
+	if runtime.GOOS == WindowsOS {
+		t.Skip("Temporarily skipping test on Windows")
+	}
```

Comment:
Can we try running it on Windows?

You'll need to update `build.proj` like:

https://github.com/pulumi/pulumi/blob/634e97cd55d8ad0347924b535f3f0daf6517a6c1/build.proj#L283-L286

and

https://github.com/pulumi/pulumi/blob/634e97cd55d8ad0347924b535f3f0daf6517a6c1/build.proj#L298